src/core/featurelistextentcontroller.cpp

```diff
@@ -62,6 +62,29 @@ void FeatureListExtentController::zoomToSelected( bool skipIfIntersects ) const
   }
 }
 
+QgsPoint FeatureListExtentController::getCentroidFromSelected() const
+{
+  QgsPoint point( 0.0, 0.0, 0.0 );
```

Comment:
Returning a 0,0,0 point is valid, we won't have any way to know whether we're actually being fed a valid centroid or not. Let's just return a QgsPoint() instead.

```suggestion
```